flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/JdbcDialect.java

```diff
@@ -66,13 +63,19 @@
      */
     String getLimitClause(long limit);
 
+    /** @return True if two instances support the same dialect. */
+    boolean equals(Object o);
```

Comment:
Where do we actually need this (same for hash code)?

Reply:
nope 🙈 that's left over from a change I didn't intend to commit. thanks for catching it!